.changeset/grumpy-islands-collect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+fix: add optional chaining back to `finalise` hook
```

Comment:
Can we describe this in a more user-friendly way?

Reply:
```suggestion
fix: preserve build error messages
```

Reply:
Sounds good to me.